vignettes/tidy-data.Rmd

```diff
@@ -37,14 +37,26 @@ Like families, tidy datasets are all alike but every messy dataset is messy in i
 Most statistical datasets are data frames made up of **rows** and **columns**. The columns are almost always labeled and the rows are sometimes labeled. The following code provides some data about an imaginary classroom in a format commonly seen in the wild. The table has three columns and four rows, and both rows and columns are labeled.
 
 ```{r}
-classroom <- read.csv("classroom.csv", stringsAsFactors = FALSE)
+library(tibble)
+classroom <- tribble(
```

Comment:
Do you mind formatting these so that the columns line up?

Reply:
of course.